specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -4245,6 +4245,14 @@
         "disableRunCommand": {
           "type": "boolean",
           "description": "Whether to disable run command for the cluster or not."
+        },
+        "enableVnetIntegration": {
```

Comment:
What happens if you set this flag to true but don't set `subnetId` below? Do they both have to be set, or do we create a subnet if the user doesn't supply one? 

If it's the second, we should say/explain that here.

Reply:
good point. thanks @matthchr .
updated its description.

Reply:
@matthchr will you review again?